Insights Docs Fixes. (#4251)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/insights/insights-setup/index.md

---
layout: layout.pug
navigationTitle: Insights Setup and Configuration
title: Insights Setup and Configuration
excerpt: Enable DKP Insights Engine
menuWeight: 20
beta: false
enterprise: false
techPreview: true
---

DKP Insights Management is installed by default on your Management Cluster.

Enable the DKP Insights Engine manually on each Attached cluster, to receive insight items on the Insights Dashboard.

## Enable the DKP Insights Engine

1.  Add DKP Insights Engine Addon

>     Apply yaml to your Management cluster from the CLI

Reviewer: Is applying this YAML file the way to accomplish what's named in the numbered step? if so, consider this wording:

Add the DKP Insights Engine Addons by applying the following YAML from the CLI:

    ```yaml
     <code>
    ```

---
layout: layout.pug
navigationTitle: Insights Setup and Configuration
title: Insights Setup and Configuration
excerpt: Enable DKP Insights Engine
menuWeight: 20
beta: false
enterprise: false
techPreview: true
---

> DKP Insights Management is installed by default on your Mangement Cluster. 

Reviewer 2: ```suggestion
DKP Insights Management is installed by default on your Management Cluster. 
```

> DKP Insights Engine needs to be enabled manually on each Attached cluster to receive insight items on the Insight Dashboard.

Reviewer 2: ```suggestion
Enable the DKP Insights Engine manually on each Attached cluster to receive insight items on the Insights Dashboard.
```

## Enable the DKP Insights Engine

> 1. Add DKP Insights Engine Addon 

Reviewer 2: ```suggestion
1. Add the DKP Insights Engine Addon 
```

>    Apply yaml to your Management cluster from the CLI

Reviewer 2: ```suggestion
   Apply yaml to your Management cluster from the CLI:
```

```bash
kubectl apply -f - <<EOF
apiVersion: source.toolkit.fluxcd.io/v1beta1
kind: GitRepository
metadata:
  name: insights-catalog-applications
  namespace: kommander
  labels:
    kommander.d2iq.io/gitapps-gitrepository-type: dkp
    kommander.d2iq.io/workspace-default-catalog-repository: "true"
spec:
  interval: 1m0s
  ref:
    branch: main
  timeout: 20s
  url: https://github.com/mesosphere/insights-catalog-applications
EOF
``` 

>    This grants all Attached clusters the ability to enable the Insights Engine, and DKP Insights icon appears under Applications.

Reviewer 2: ```suggestion
   This grants all Attached clusters the ability to enable the Insights Engine. The DKP Insights icon appears under Applications.
```

> 2. Enable the DKP Insights Engine on each Attached cluster from the DKP UI by selecting: Cluster -> Applications -> Options Menu -> Enable -> Deploy with defaults.

Reviewer 3: Yea, that's better. good catch, I was going to get back to this and forgot about it!
